### PR TITLE
Database split. `KC` uses its own database

### PR DESCRIPTION
**Pull request by noliveleger**
Has to be merged with: 

Needs to be merged with:

- kobo-docker https://github.com/kobotoolbox/kobo-docker#230
- kobo-install https://github.com/kobotoolbox/kobo-install/pull/15
- kpi https://github.com/kobotoolbox/kpi/pull/2222

A management command must be run on servers to update DB.
`./manage.py update_kpi_hooks_endpoint`

### CODE REVIEW

File: onadata/libs/utils/middleware.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import traceback
 
+from django.contrib.auth.middleware import AuthenticationMiddleware
```
Comment (jnm) [inline onadata/libs/utils/middleware.py]: It seems like this import isn't used (but check to be sure?)

File: README.rst [hunk 2]
```diff
@@ -1,3 +1,9 @@
+Important notice when upgrading from ``[TODO: INSERT FINAL 1DB RELEASE HERE]`` or earlier
+-----------------------------------------------------------------------------------------
+
+Up to and including release ``[TODO: INSERT FINAL 1DB RELEASE HERE]``, this project (KoBoCAT) and `KPI <https://github.com/kobotoolbox/kpi>`_ both shared a common Postgres database. They now each have their own. **If you are upgrading an existing single-database installation, you must follow** `these instructions <TODO: LINK>`_ to migrate the (smaller) KPI tables to a new database and adjust your configuration appropriately.
```
Comment (awemulya) [inline README.rst]: #TODO Link missing

Comment (tolexy) [reply README.rst]: Quick question: will the ToDo link get updated/completed in the semi near-future or is there any general timeline for that?

Comment (noliveleger) [reply README.rst]: @tolexy, it is updated now

**Pull request merged**